ssg/build_yaml.py

```diff
@@ -131,6 +131,16 @@ def to_xml_element(self):
 
         return element
 
+    def get_rules(self):
```

Comment:
`get_rules` can give impression that it returns all rules from profile, while it only returns explicitly selected rules.
The rules selected by "extension" are not returned.
My suggestion: `get_rule_selectors`

Reply:
Fixed